prow/jobs/custom/prow-autobump.yaml

```diff
@@ -1,88 +0,0 @@
-# Copyright 2022 The Knative Authors
```

Comment:
I sorta liked having the job separated out. It would be nice to actually split up test-infra.yaml into more bite size chunks in general. However, I might not understand the implication of having a bunch of custom jobs vs a monolithic yaml.